tensorflow/core/common_runtime/constant_folding_test.cc

```diff
@@ -687,8 +684,10 @@ class TestTFFileSystem : public ::tensorflow::NullFileSystem {
       : ::tensorflow::NullFileSystem(),
         data_tensor_(test::AsTensor<double>({1., 2., 3., 4.}, {2, 2})) {}
 
+  using ::tensorflow::NullFileSystem::NewReadOnlyMemoryRegionFromFile;
+
```

Comment:
We don't need this if we have the macro, right?

Reply:
If we use this, we only expose the overridden method form NullFileSystem. The reason I did that, if at some point in time NullFileSystem is modified, with the macro, these modifications will not be visible in this class. With this approach, NullFileSystem implementations will be used instead of FileSystem base class implementations.